### ISSUE TITLE
Elemental 3 compatibility

### ISSUE THREAD

**Issue by DorsetDigital** [dnadesign/silverstripe-elemental-userforms#35]
With the current version of Elemental being at 3.x, I'm just wondering if there's a reason not to update the requirements of this element.  Having it pegged at Elemental 2 is starting to cause dependency issues when using other element add-ons.

I've done some testing against Elemental 3 and haven't seen any issues so far.

**Comment by robbieaverill** [comment]
Yeah 2 and 3 should be basically the same, I think we changed the CMS UI and broke something in terms of semver, but we should be able to allow the constraint for it safely enough
